README.md

```diff
@@ -17,22 +36,58 @@ keytool -genkey -alias selfsigned -keyalg RSA -keystore src/main/docker/etc/conf
 **Make sure the key password and store password are the same!** This is a requirement for Spring Security.
 
 2. Then, make sure [Docker][] and [Docker-Compose][] are installed on your system.
-3. Finally, we can start the stack with `docker-compose -f src/main/docker/app.yml up -d`.
+3. Finally, we can start the stack with `docker-compose -f src/main/docker/management-portal.yml up -d`.
```

Comment:
Docker setup is optional for MP. It should be covered under docker.

Reply:
The Dockerfile under docker folder doesn't do the same as the one under the root directory. It in fact doesn't work as expected. If we have to fully rely on test automation and CI we have, I would recommend having a single dockerfile and make sure it works as expected in all cases.

Reply:
The Dockerfile in the docker folder is meant for the gradle `buildDocker` task and its purpose is to build an image based on the war generated with `bootRepackage`, i.e. a quick build from an existing compiled file.

The Dockerfile in the project root is meant for Dockerhub automated builds. Since the pre-built war is not available in that scenario this Dockerfile includes the compilation step.

I agree the Readme in general could benefit from some restructuring. Should we modify the quickstart section to explain using `gradlew bootRun`, i.e. `dev` profile with demo data; and have a separate section for all the Docker related information, i.e. `prod` profile and associated configuration?